test/plugins/features/movies/controller.test.js

```diff
@@ -23,14 +43,67 @@ describe('movie controller', () => {
 
   });
 
-  describe('get', () => {
+  describe('list', () => {
+
+    it('list movies released after year_start', () => {
+      const payload = { year_start: 1983 };
+
+      return Controller.list(payload)
+      .then((json) => {
+        json.map((movie) => {
+          expect(movie.attributes.release_year).to.be.at.least(1983);
```

Comment:
Nit: usually to access properties of the model, you should use `.get`: `expect(movie.get('release_year'))...`